www/js/app.js

```diff
@@ -1034,6 +1076,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'cookies','abstractFilesystemAcc
      * @param {DirEntry} dirEntry The directory entry of the article to read
      */
     function readArticle(dirEntry) {
+        cookies.setItem("lastReadDir", JSON.stringify(dirEntry), Infinity);
```

Comment:
@Hardik500 I would call this cookie item "lastReadDirEntry" to be clearer on what is being stored.